cloud/aws/bin/deploy.py

```diff
@@ -0,0 +1,26 @@
+#! /usr/bin/env python3
+import subprocess
+import os
+import sys
+
+sys.path.append(os.getcwd())
+
+from cloud.shared.bin.lib import civiform_mode
+from cloud.shared.bin.lib import terraform
+
+# Keep in sync with cloud/shared/bin/lib.sh
+os.environ['TF_VAR_FILENAME'] = 'setup.auto.tfvars'
+os.environ['BACKEND_VARS_FILENAME'] = 'backend_vars'
+
+subprocess.check_call(['cloud/shared/bin/lib/tf_apply_setup.py'])
```

Comment:
No changes requested, just noting for posterity:

While `.run()` is apparently the preferred way to call `subprocess` nowadays. That said, it defaults the `check` arg to False, which feels like a pretty unsafe default to me.

Personally, I prefer `check_call` since it's more explicit about what's happening.